src/sync/lookupApiUtils.js

```diff
@@ -237,7 +237,7 @@ export const getPatientHistoryResponseProcessor = ({
           doses: totalQuantity * doses,
           itemCode,
           itemName,
-          prescriber,
+          prescriber: !isVaccine && vaccinator ? vaccinator : prescriber,
```

Comment:
shouldn't that be `isVaccine && vaccinator`?

Reply:
🤣  I considered renaming `isVaccine` - maybe I should have! It looks like it's used to determine whether we are on the Dispensing history or the Vaccine Dispensing history? So `!isVaccine` actually means we are on Dispensing History

Reply:
oh 🤦  I see now. thanks.